app/src/main/java/org/p2p/wallet/swap/jupiter/interactor/JupiterSwapInteractor.kt

```diff
@@ -37,13 +35,14 @@ class JupiterSwapInteractor(
             route = route,
             userPublicKey = tokenKeyProvider.publicKey.toBase58Instance()
         )
-        val latestBlockhash = rpcBlockhashRepository.getRecentBlockhash()
         val userAccount = Account(tokenKeyProvider.keyPair)
 
         val signedSwapTransaction = relaySdkFacade.signTransaction(
             transaction = swapTransaction,
             keyPair = userAccount.getEncodedKeyPair().toBase58Instance(),
-            recentBlockhash = latestBlockhash
+            // empty string because swap transaction already has recent blockhash
+            // if pass our own recent blockhash, there is an error
+            recentBlockhash = null
```

Comment:
but you are cleaning the current blockhash by this, no?

Reply:
no, the current blockhash in transaction already exists from Jupiter, and our SDK overwrites it that causes error from RPC